### PR TITLE
tracing: use ints instead of buffers

### PR DESCRIPTION
**Pull request by rf**
r @Raynos @jcorbin

### CODE REVIEW

File: test/tcollector-reporter.js [hunk 1]
```diff
@@ -152,9 +152,9 @@ test('jsonSpanToThriftSpan', function t2(assert) {
     var mapped = TCollectorReporter.jsonSpanToThriftSpan(testSpan);
 
     assert.equals(mapped.name, testSpan.name);
-    assert.deepEquals(mapped.traceId, testSpan.traceid);
-    assert.deepEquals(mapped.id, testSpan.id);
-    assert.deepEquals(mapped.parentId, testSpan.parentid);
+    assert.deepEquals(mapped.traceId, Buffer([0, 0, 0, 0, 0, 0, 0, 1]));
```
Comment (jcorbin) [inline test/tcollector-reporter.js]: why for these buffers still?

**Comment by jcorbin** [comment]
I like less buffers, and you updated the tests, but I'm a bit confused why still buffers?

File: test/tcollector-reporter.js (hunk 1 above)
Comment (Raynos) [inline test/tcollector-reporter.js]: Let's use the thrift long support once its not broken.

Comment (rf) [inline test/tcollector-reporter.js]: These are about to be thrift serialized, and in the thrift file these are still `binary`. `jsonSpanToThriftSpan` converts a span from the internal TChannel representation to something ready to be sent to the `tchannelThrift.send` call.

**Comment by Raynos** [comment]
Please Long(low, high) instead of [high, low].

Let's add a dep on Long.

**Comment by rf** [comment]
These aren't Longs, they're 64 random bits. Using Long to represent them is unnecessary complexity. Also, they're `binary` in the thrift file, not i64s.

**Comment by Raynos** [comment]
I see. Hmm. stop using arrays and have double the fields XD

File: test/tcollector-reporter.js (hunk 1 above)
Comment (Raynos) [inline test/tcollector-reporter.js]: Oh the thrift is binary; I guess we have to buffers. #coolstory

**Comment by Raynos** [comment]
lgtm. Ill stop being annoying.

**Comment by kriskowal** [comment]
We can add a `binary (js.type = 'Long')` annotation to thriftrw if needs be.

**Pull request merged**